### ISSUE TITLE
MozFest: Logo and Site Nav 

### ISSUE THREAD

**Issue by mmmavis** [mozilla/foundation.mozilla.org#3206]
TL;DR - Do we want the height of the primary nav section to be the same as what's on the main foundation site?

---

I followed the specs in Sketch file to set the dimension of the MozFest logo used in nav (99px * 37px).  Currently we make the primary nav on MozFest site to follow what's on the main Foundation site style (e.g., spacing etc). Since the dimensions of the two logos are different, this creates a few issues.

For example, floating underline on hover

![Artboard 1](https://user-images.githubusercontent.com/2896608/58140858-df981200-7bf5-11e9-83e1-f2c41a91f376.png)

MozFest logo is bigger and taller. This makes the nav taller as well and hence adds extra space below the link underline. We can easily fix this issue via different approaches. But I just need to know if we wanna keep the nav height the same or not & if we need to make adjustment to the size of the logo and the default padding (highlighted in purple) of the nav.

<img width="1151" alt="image" src="https://user-images.githubusercontent.com/2896608/58140756-541e8100-7bf5-11e9-8ab0-c036f468ade9.png">

/cc @sabrinang

**Comment by kristinashu** [comment]
The idea is that the height of the nav is the same as on the foundation site. And it looks like the mozfest logo is the same width as the mozilla logo, but is taller. Is that ok to implement?

**Comment by mmmavis** [comment]
Their sizes are actually different (on prod and on Sketch as well).
<img width="210" alt="Screen Shot 2019-05-22 at 12 29 09 PM" src="https://user-images.githubusercontent.com/2896608/58202752-38ff5000-7c8d-11e9-93fe-ad29b1dbc94b.png">
<img width="223" alt="Screen Shot 2019-05-22 at 12 29 01 PM" src="https://user-images.githubusercontent.com/2896608/58202756-3a307d00-7c8d-11e9-96aa-9ebf3fa7ddf0.png">

Should I resize MozFest logo so it shows up as 97px wide and whatever its corresponding height is?

**Comment by sabrinang** [comment]
Yes, I tried to match the existing width on Sketch (which I didn't know is diff from prod) so please match the 97px width similar to mozilla logo on prod.
